std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
I just see no need to scope std.range.primitives imports, they are ubiquitous. The whole point of these free standing functions is _seemless_ implementation of range interface, I find it backwards for function to accept a "range" (all 3 primitives) and then import empty or front/popFront.